### PR TITLE
Additional onboarding support for loop status colors and dosing enabled

### PR DESCRIPTION
**Pull request by darinkrauss**

### CODE REVIEW

**Comment by darinkrauss** [comment]
LW PR for build: https://github.com/tidepool-org/LoopWorkspace/pull/466

**Review by nhamming — APPROVED** [review]
LGTM

File: LoopKitUI/Extensions/Environment+Colors.swift [hunk 1]
```diff
@@ -67,6 +69,20 @@ public extension EnvironmentValues {
     }
 }
 
+private struct LoopStatusColorPaletteKey: EnvironmentKey {
+    static let defaultValue: StateColorPalette = StateColorPalette(unknown: .systemGray4,
+                                                                   normal: .green,
+                                                                   warning: .orange,
```
Comment (ps2) [inline LoopKitUI/Extensions/Environment+Colors.swift]: Nit: I think default warning for LoopKit should be yellow.

**Review by ps2 — APPROVED** [review]
LGTM

**Pull request merged**